CONTRIBUTING.md

```diff
@@ -0,0 +1,213 @@
+<!--
+SPDX-FileCopyrightText: � 2021-2022 MONAI Consortium
+ SPDX-License-Identifier: Apache License 2.0
+-->
+- [Introduction](#introduction)
+  - [Communicate with us](#communicate-with-us)
+- [The contribution process](#the-contribution-process)
+  - [Preparing pull requests](#preparing-pull-requests)
+  - [Submitting pull requests](#submitting-pull-requests)
+  - [Release a new version](#release-a-new-version)
+
+
+## Introduction
+
+Welcome to Project MONAI Deploy Informatics Gateway! We're excited you're here and want to contribute. This documentation is intended for individuals and institutions interested in contributing to the MONAI Deploy Informatics Gateway. MONAI Deploy Informatics Gateway is an open-source project. As such, its success relies on its community of contributors willing to keep improving it. Therefore, your contribution will be a valued addition to the code base; we ask that you read this page and understand our contribution process, whether you are a seasoned open-source contributor or a first-time contributor.
```

Comment:
@mocsharp this needs some text updates to refer to the workflow manager instead of the Informatics Gateway